tests/test.c

```diff
@@ -326,6 +327,7 @@ struct test tests[] = {
         _TEST(0098_consumer_txn, 0),
         _TEST(0099_commit_metadata, 0),
         _TEST(0100_thread_interceptors, TEST_F_LOCAL),
+        _TEST(0101_fetch_from_follower, 0),
```

Comment:
specify the minimum broker version here

Reply:
oh cool, missed that feature

Reply:
... but a bit inconvenient currently as this doesn't work in the `./interactive_broker_version.py trunk` env.